[buildout]

extends =
    https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.2.x.cfg
    https://raw.githubusercontent.com/collective/buildout.plonetest/master/qa.cfg
    base.cfg

[versions:python27]
lazy-object-proxy = 1.6.0

# FIXME: waitress = 2.1.2 fix the error:
#     OSError: [Errno 9] Bad file descriptor
# See: https://github.com/Pylons/waitress/issues/374
# This error causes the server to "freeze" on robot tests and the tests never finish.
# waitress = 2.1.2 has been pinned in Zope 4.8.2. So this pinn here should be removed
# when Plone 5.2.9 is released.
[versions:python37]
waitress = 2.1.2

[versions:python38]
waitress = 2.1.2
